### ISSUE TITLE
Update contents programatically?

### ISSUE THREAD

**Issue by DylanPiercey** [yabwe/medium-editor#405]
Is it possible to update the contents of a medium-editor programatically? (Without finding the element and overriding the html manually).

**Comment by hellocosmin** [comment]
Yes, though you need to do it yourself. You need to deactivate(), set the new properties for whatever options you want to change and activate() back.

See https://github.com/daviferreira/medium-editor/issues/385 for an example with dynamic buttons.

**Comment by daviferreira** [comment]
@DylanPiercey the problem is what to do when you have multiple editors using the same selector. Right now you can use the insertHTML method, but it will update the focused element only.

**Comment by nmielnik** [comment]
It seems like we would just want a helper method that allows you to set the content of the editor's element(s).

Something like:

```js
setContent: function (html, index) {
    this.elements[index].innerHTML = html;
    // We probably want to default index to either 0, or to loop through all elements and set the content for all elements
}
```

**Comment by nmielnik** [comment]
As pointed out in #731 we would want this code to also trigger the `editableInput` event so any listeners would know about the change to the content.

**Comment by nmielnik** [comment]
Fixed by #736

**Comment by gsouf** [comment]
Hi, is there an alternative to setContent that does not trigger ``editableInput`` event?
